x-pack/plugins/maps/public/routes/map_page/map_app/map_app.tsx

```diff
@@ -163,7 +215,16 @@ export class MapApp extends React.Component<Props, State> {
       return;
     }
 
-    this._onQueryChange({ time: globalState.time });
+    if (changes.time) {
+      this._onQueryChange({ time: globalState.time });
+    }
+
+    if (changes.refreshInterval && globalState.refreshInterval) {
```

Comment:
Do we need both `changes.refreshInterval && globalState.refreshInterval` for this condition? On line 218 we're just looking at `changes.time`

Reply:
I will updated the time check to include check that globalState.time exists.